tests/conftest.py

```diff
@@ -67,9 +67,11 @@ def filter_access_token(interaction, current_cassette):
     x: env_default(x)
     for x in (
         "auth_code client_id client_secret password redirect_uri "
-        "test_subreddit user_agent username"
+        "test_subreddit user_agent username refresh_token"
```

Comment:
The change to add support for `refresh_token` seems pretty simple, which is good. I do have a few questions about it.

1. Is there any way that this change could be separated out into a new PR, since it isn't directly related to the moderation streams feature?
2. If you record a test's cassette with a refresh token, would I be able to re-record the cassette with a username and password as I would any other (i.e. just by deleting the cassette and running the tests with my username and password as environment variables)? And, in reverse, would you be able to re-record any username/password cassette using a refresh token? (This is a question I'd rather ask in a PR discussing the refresh token support specifically, so if the answer to question number 1 is yes, then there's no need to answer or discuss this question here)

Reply:
1. With the cassettes I recoded, no. These cassettes were recoded with a refresh token. See comment above for why I used the refresh token.
2. Yes, because when the tests that use token auth are tested it init's its own reddit instance using token auth with the placeholders. It will work in reverse too because nothing is done with the refresh_token when `setup_reddit` is called before the tests. It will just use username/password with place holders. All the tests pass with refresh_token, username/password, neither when I run them.